infrastructure/helpers/lit-tests.yaml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Run Mull examples
```

Comment:
How can I run just this helper file to iterate on it locally?

Reply:
Exactly as described in the scenario:
```
cd $SOURCE_DIR/tests-lit
export CLANG_EXEC=$LLVM_DIR/bin/clang++
make run_cxx
```